packages/node_modules/samples/browser-meetings/index.html

```diff
@@ -117,32 +117,43 @@ <h2>Meetings Management</h2>
 
     <section>
       <h2>Meeting Controls</h2>
-      <p>[[MEETING CONTROLS DESCRIPTION]]</p>
+      <p>Once in a meeting utilize these controls to manipulate the meeting.</p>
       <form id="general-controls">
-        <fieldset>
+        <fieldset disabled>
+          <legend>Selected meeting</legend>
+          <div>        
+            <select id="gc-meetings-list">
+                <option selected>No meetings available</option>
+            </select>
+            <p>Use the dropdown to select which meeting to manipulate.</p>
+          </div>
+        </fieldset>
+        <fieldset disabled>
           <legend>General Controls</legend>
```

Comment:
If you could, add a `<p />` element below this `<legend />` element that indicates that the meeting must be joined as a moderator in order to utilize the controls in this subsection.

Reply:
That's a a great idea, will do.